pyvista/utilities/__init__.py

```diff
@@ -1,38 +1,69 @@
-"""Deprecated utilities module."""
+"""Deprecated utilities subpackage."""
 import importlib
 import inspect
 import warnings
 
-from pyvista.core.errors import DeprecationError, PyVistaDeprecationWarning
+# Places to look for the utility
+_MODULES = [
+    'pyvista.core.utilities',
+    'pyvista.core.errors',
+    'pyvista.core',
+    'pyvista.plotting.utilities',
+    'pyvista.plotting.errors',
+    'pyvista.plotting.texture',
+    'pyvista.plotting',
+    'pyvista.report',
+    'pyvista.themes',
+]
 
 
-def __dir__():
-    raise DeprecationError('The `pyvista.utilities` module has been deprecated.')
+def _try_import(module, name):
+    """Attempt to import a module."""
+    _module = importlib.import_module(module)
+    try:
+        feature = inspect.getattr_static(_module, name)
+        import_path = f'from {module} import {name}'
+    except AttributeError:
+        return None, None
+    return feature, import_path
 
 
 def __getattr__(name):
-    utils = importlib.import_module('pyvista.core.utilities')
+    """Fetch an attribute ``name`` from ``globals()`` and warn if it's from a deprecated module.
+
```

Comment:
In case you agree that this helps (so that nobody (like me) thinks later "surely we don't need `globals()` in there!"):
```suggestion

    Note that ``__getattr__()`` only gets called when ``name`` is missing
    from the module's globals. The trick is that we want to import this
    function into other deprecated modules, and we want to carry this
    subpackage's globals along to prevent some spurious warnings.

```
Phrasing open for discussion, of course.